modules/auxiliary/scanner/misc/rocketmq_version.rb

```diff
@@ -0,0 +1,104 @@
+### This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache RocketMQ Version Scanner',
+        'Description' => %q{
+          Version scanner for the Apache RocketMQ product.
+        },
+        'Author' => [
+          'h00die',
+          'Malayke' # CVE-2023-33246 code
+        ],
+        'References' => [
+          ['URL', 'https://github.com/Malayke/CVE-2023-33246_RocketMQ_RCE_EXPLOIT/blob/main/check.py'],
+          ['URL', 'https://github.com/apache/rocketmq']
+        ],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [],
+          'Reliability' => [],
+          'SideEffects' => []
+        }
+      )
+    )
+    register_options([ Opt::RPORT(9876), ])
+  end
+
+  def get_version(id)
+    # from https://github.com/apache/rocketmq/blob/develop/common/src/main/java/org/apache/rocketmq/common/MQVersion.java
```

Comment:
```suggestion
    # from https://github.com/apache/rocketmq/blob/develop/common/src/main/java/org/apache/rocketmq/common/MQVersion.java
```

https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files#press-y-to-permalink-to-a-file-in-a-specific-commit